error_reporting/unit_tests/test_client.py

```diff
@@ -128,6 +128,32 @@ def test_report(self):
         self.assertLess(report_location['lineNumber'], 150)
 
 
+class TestFlaskHelper(unittest.TestCase):
```

Comment:
1. The name should reflect what is being tested: `Test_build_flask_context`
1. The fact that this test is in `test_client` indicates that the function being tested is in `client.py`, which is not true. So either move the function or create a new `test___init__.py` module (I prefer the first choice)